doc/Caliptra.md

```diff
@@ -925,38 +927,38 @@ A remote verifier wishes to confirm that a given Caliptra device has not run ver
 2. An LDevID-signed certificate over Alias<sub>FMC</sub>, which includes FMC's measurements as captured by ROM.
 3. An Alias<sub>FMC</sub>-signed certificate over Alias<sub>RT</sub>, which includes runtime firmware's measurements as captured by FMC.
 4. An Alias<sub>RT</sub>-signed certificate over a leaf DPE key, which includes PCR0 as read by runtime firmware, along with other measurements previously stashed in SRAM.
```

Comment:
s/PCR0/PCR3/, throughout this list.

Reply:
should be fixed with merge conflict resolution but please open issue if needed.